packages/config/config/devices/0x0234/templates/logic_group_template.json

```diff
@@ -0,0 +1,560 @@
+		"label": "Function setup of input",
+		"maxValue": 3,
+		"defaultValue": 3,
+		"allowManualEntry": false,
+		"options": [
+			{
+				"label": "Disabled",
+				"value": 0
+			},
+			{
+				"label": "Level-controlled input",
+				"value": 1
+			},
+			{
+				"label": "Falling-edge controlled input",
+				"value": 2
+			},
+			{
+				"label": "Rising-edge controlled input",
+				"value": 3
+			}
+		]
+	},
+	"output_function_relay": {
+		"$import": "../../templates/master_template.json#base_enable_disable",
+		"label": "Output function",
+		"options": [
+			{
+				"label": "Controlled via Z-Wave messages",
+				"value": 0
+			},
+			{
+				"label": "Controlled by input 1",
+				"value": 1
+			}
+		]
+	},
+	"status_led": {
+		"$import": "../../templates/master_template.json#led_indicator_four_options",
+		"label": "Status of LED",
+		"defaultValue": 1,
+		"options": [
+			{
+				"label": "Disabled",
+				"value": 0
+			},
+			{
+				"label": "Enabled",
+				"value": 1
+			},
+			{
+				"label": "Flashes in 1 second interval",
+				"value": 2
+			},
+			{
+				"label": "Flashes in 1/2 second interval",
+				"value": 3
+			}
+		]
+	},
+	"meter_type": {
+		"label": "Meter Type of input",
+		"description": "Meter Type of input",
+		"valueSize": 1,
+		"minValue": 0,
+		"maxValue": 5,
+		"defaultValue": 1,
+		"readOnly": false,
+		"writeOnly": false,
+		"allowManualEntry": false,
+		"options": [
+			{
+				"label": "Input not in use",
+				"value": 0
+			},
+			{
+				"label": "Electricity meter",
+				"value": 1
+			},
+			{
+				"label": "Gas meter.",
+				"value": 2
+			},
+			{
+				"label": "Water meter.",
+				"value": 3
+			},
+			{
+				"label": "Heat meter",
+				"value": 4
+			},
+			{
+				"label": "Cooling meter.",
+				"value": 5
+			}
+		]
+	},
+	"input_scale": {
+		"label": "Scale Type of input",
+		"valueSize": 1,
+		"minValue": 0,
+		"maxValue": 6,
+		"defaultValue": 3,
+		"unsigned": true,
+		"readOnly": false,
+		"writeOnly": false,
+		"allowManualEntry": true
+	},
+	"input_precision": {
+		"label": "Precision of input",
+		"valueSize": 1,
+		"minValue": 0,
+		"maxValue": 7,
+		"defaultValue": 0,
+		"unsigned": true,
+		"readOnly": false,
+		"writeOnly": false,
+		"allowManualEntry": true
+	},
+	"pulse_unit": {
+		"label": "Pulses per Unit of input 5",
+		"unit": "pulse",
+		"valueSize": 2,
+		"minValue": 0,
+		"maxValue": 65000,
+		"defaultValue": 1,
+		"unsigned": true,
+		"readOnly": false,
+		"writeOnly": false,
+		"allowManualEntry": true
+	},
+	"rate_type": {
+		"label": "Rate Type of input",
+		"valueSize": 1,
+		"minValue": 0,
+		"maxValue": 3,
+		"defaultValue": 1,
+		"unsigned": true,
+		"readOnly": false,
+		"writeOnly": false,
+		"allowManualEntry": false,
+		"options": [
+			{
+				"label": "Unspecified",
+				"value": 0
+			},
+			{
+				"label": "Import",
+				"value": 1
+			},
+			{
+				"label": "Export",
+				"value": 2
+			},
+			{
+				"label": "Reserved",
+				"value": 3
+			}
```

Comment:
If this parameter does what I think (Meter CC Rate Type), this option shouldn't be offered.

Reply:
Removed from the code although this value is in their documentation.